admin/app/controllers/cache/ImageDecacheController.scala

```diff
@@ -40,10 +39,11 @@ class ImageDecacheController extends Controller with Logging with AuthLogging wi
 
       val originUri = new URI(originUrl)
 
-      ImageServices.clearFastly(originUri)
-      ImageServices.clearImgix(originUri)
+      val imageServices = new ImageServices(wsClient)
```

Comment:
Good point. I don't think it is used a lot but I will extract the creation outside of the method